index.js

```diff
@@ -0,0 +1,54 @@
+const form = document.querySelector('form');
+const bookList = document.getElementById('book-list');
+const bookArr = [];
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const obj = {
+    title: document.getElementById('book_title').value,
+    author: document.getElementById('author').value,
+  };
+
+  if (obj.title === '' || obj.author === '') {
+    e.preventDefault();
+  } else if (localStorage.length === 0) {
+    bookArr.push(obj);
+    localStorage.setItem('bookArrObj', JSON.stringify(bookArr));
+
+    location.reload();
+  } else {
+    const ArrayStored = localStorage.getItem('bookArrObj');
+    const ArrayStoredParse = JSON.parse(ArrayStored);
+    ArrayStoredParse.push(obj);
+    localStorage.setItem('bookArrObj', JSON.stringify(ArrayStoredParse));
+
+    location.reload();
+  }
+});
```

Comment:
When I try to submit a new Book with `book title` and `author name` it's not getting the values properly and it's not submitting the new book and rendering to the page. Kindly fix this functionality. Below I attached the error:

<img width="1676" alt="image" src="https://user-images.githubusercontent.com/34578631/163971764-e9ee1956-f668-4d81-a470-4cf542a3d089.png">

Reply:
HI @Mre55,
The project working perfectly on my side, I don't know what's happened with your browser
Can you try please in another one like firefox

Reply:
Hi @camilevahviraki,

I see it is working in Firefox perfectly and also I notice after your last change now it's also working in Google chrome. Please ask for re-reviews the next reviewer will see the changes.